app/models/king.rb

```diff
@@ -14,4 +14,37 @@ def valid_move?(x, y)
   def radial_move?(x, y)
     x_distance(x) <= 1 && y_distance(y) <= 1
   end
+
+  # Returns true if the king and rook have not moved,
+  # the location is a valid castling target, and there is
+  # no obstruction along the way. DOES NOT INCLUDE CHECK LOGIC.
+  def clear_castling_move?(x, y)
```

Comment:
IMO this name is ambiguous. Maybe instead `is_castle_target?` or `valid_castle_target?`